app/controllers/sessions_controller.rb

```diff
@@ -5,7 +5,10 @@ def new
 
   def create
     user = User.find_by(username: params[:username])
-    if user && user.authenticate(params[:password])
+    if user && user.authenticate(params[:password]) && user.admin?
```

Comment:
Think we could change to 'if user.authenticate(params[:password]) && user.admin?' since can only be true if user isn't nil